vms/proposervm/block.go

```diff
@@ -82,9 +82,9 @@ func (p *postForkCommonComponents) Verify(parentTimestamp time.Time, parentPChai
 	}
 
 	childPChainHeight := child.PChainHeight()
-	if childPChainHeight < parentPChainHeight {
-		return errPChainHeightNotMonotonic
-	}
+	//if childPChainHeight <= parentPChainHeight { // Since p-chain height doesn't increase for Flare
+	//	return errPChainHeightNotMonotonic
+	//}
```

Comment:
I'd rather remove the code completely than comment it out if that's ok with you?